src/hal/src/lib.rs

```diff
@@ -15,6 +15,10 @@ extern crate mint;
 #[macro_use]
 extern crate serde;
 
+#[cfg(feature = "hlist-spec-const")]
+#[doc(hidden)]
+pub use frunk_core;
```

Comment:
does it need to be pub?

Reply:
No. And I'm getting rid of the dependency anyway ;)